### PR TITLE
Creating Unitals file and moving neccessary data

### PR DESCRIPTION
**Pull request by Awurama-N**
moving content from 131 onwards to new file "Unitals"
closes issue #2394

### CODE REVIEW

**Comment by Awurama-N** [comment]
when i ran make i didn't come across issues related to the Unitals file. But there's a warning here that the module cannot be found. please How can I fix that?

File: code/drasil-example/Drasil/PDController/DataDefs.hs [hunk 1]
```diff
@@ -4,6 +4,8 @@ import Drasil.PDController.Assumptions
 
 import Drasil.PDController.Concepts
 
+import Drasil.PDcontroller.Unitals
```
Comment (JacquesCarette) [inline code/drasil-example/Drasil/PDController/DataDefs.hs:7]: while you're at it, can you delete all those extra blank lines in the import list please?

File: code/drasil-example/drasil-example.cabal [hunk 2]
```diff
@@ -215,6 +215,7 @@ executable pdController
       Drasil.PDController.Requirements
       Drasil.PDController.SpSysDesc
       Drasil.PDController.TModel
+      Drasil.PDcontroller.Unitals
```
Comment (JacquesCarette) [inline code/drasil-example/drasil-example.cabal:218]: This is going to get automatically generated - it is better not to add it by hand (unlike for `package.yaml`)

**Review by JacquesCarette — CHANGES_REQUESTED** [review]

**Comment by Awurama-N** [comment]
okay will do. is there anything else i have to do to stop the build check from failing?

File: code/drasil-example/Drasil/PDController/Body.hs [hunk 3]
```diff
@@ -71,6 +71,7 @@ import Language.Drasil.Code
 import Language.Drasil.Printers (PrintingInformation(..), defaultConfiguration)
 import Theory.Drasil (DataDefinition, GenDefn, InstanceModel, TheoryModel)
 import Utils.Drasil
+import Drasil.PDcontroller.Unitals
```
Comment (JacquesCarette) [inline code/drasil-example/Drasil/PDController/Body.hs:74]: This should be `PDController`, not `PDcontroller` (capital C, not lower-case c). This occurs multiple times - that's the reason it cannot be found.

**Review by JacquesCarette — CHANGES_REQUESTED** [review]

File: code/drasil-example/Drasil/PDController/Body.hs (hunk 3 above)
Comment (Awurama-N) [reply code/drasil-example/Drasil/PDController/Body.hs:74]: oh okay

**Pull request closed without merging**